app/assets/stylesheets/base/org_types.scss

```diff
@@ -0,0 +1,85 @@
+
+// NOTE: THESE ORG TYPE BG AND TEXT COLORS AREN'T ACCURATE. needs to be reviewed. 
+$animals:        #22aa99;
+$community:      #ff88aa;
+$education:      #1199ee;
+$environment:    #99cc66;
+$health:         #7755cc;
+$human_rights:   #4267b2;
+$human_services: #EA958C;
+$international:  #f3c937;
+$religion:       #F65633;
```

Comment:
don't we have a variable.scss file some where?? we should keep it centralized

Reply:
if not, can we have like a `_variables.scss`?? also I'm more a fan of what these colors are instead of the the actual org, and maybe use what you have in `$org_type_colors` to refer to the org name.

So for example like `"animals" : $dark_green`

Reply:
or like `$gp_green` (shortening giving profiles to `gp`)

This is so that readers can see what the hex color actually is instead of looking it up

Reply:
> don't we have a variable.scss file some where?? we should keep it centralized
I think Rashad made that on his branch. I don't really do CSS much so I'm not not too familiar with what the structure should be. Are you suggesting to put ALL variables from ALL parts of the app into one file?

Reply:
Yes, here's a pretty good guideline for sass https://www.sitepoint.com/architecture-sass-project/